instrumentation/jaxws/jaxws-2.0-cxf-3.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/cxf/CxfHelper.java

```diff
@@ -31,8 +29,7 @@ public static void start(Message message) {
       return;
     }
 
-    ServerSpanNaming.updateServerSpanName(
-        parentContext, CONTROLLER, CxfServerSpanNaming.SERVER_SPAN_NAME, request);
+    CxfServerSpanNaming.updateServerSpanName(parentContext, request);
```

Comment:
One problem with not using `ServerSpanNaming.updateServerSpanName` is that when there is an exception and request is dispatched to exception handling servlet it can overwrite the name set by the original request. `ServerSpanNaming` avoided it by disallowing name update when new name had the same or lower priority as the name set.

Reply:
Does that really happen though? We don't really call `updateServerSpanName` at the end of processing; there's one invocation in `finally` block in `OpenTelemetryHandlerMappingFilter` in spring-webmvc, but it's a web controller framework - it won't run jax-ws code in a controller (probably).

Reply:
In web.xml you can set error pages where container will dispatch when there is an exception or when request status is 500 or whatever. This error handler can be a servlet and could potentially also try to update server span name.

Reply:
True, that's a valid scenario. Hmm, is this behavior that important, should we keep it? Or would it be fine to just ignore that case (since it's just jax-ws; and maybe jax-ws has its own exception handler interface or something)

Reply:
I believe that for jax-ws it is ok to ignore it. It needs to send errors back as a soap response so it can't use servlet error handling.